### PR TITLE
2017 tasks: Update Interactive bots - Task A

### PR DESCRIPTION
**Pull request by abhijeetkaur**
- [ ] Update link that points to "Task for submitting pull request"

### CODE REVIEW

File: tasks/2017/interactive-bots.md [hunk 1]
```diff
@@ -35,92 +35,56 @@ messages and can be integrated with various APIs.
...
-  use the bot in.
+* Create a bot with the name `<your-name>_helloworld_bot`.
+(For details refer to the second step [here](https://chat.zulip.org/api/running-bots))
 
-* Download your bot's `zuliprc` file, by clicking on the green button, and
-save it as `~/zuliprc-local`.
+* Since we aim to run a "helloworld" bot, replace <my_bot> with "helloworld" while [running a bot](https://chat.zulip.org/api/running-bots).
 
-  Alternatively, you can create this file with credentials:
+* Use `Manage Streams` to subscribe your bot (`<your-name>_helloworld_bot`) to any stream
```
Comment (derAnfaenger) [inline tasks/2017/interactive-bots.md]: We don't need to subscribe bots to streams anymore, so this step can be removed.

File: tasks/2017/interactive-bots.md [hunk 2]
```diff
@@ -35,92 +35,56 @@ messages and can be integrated with various APIs.
...
-
-
-* Test manually that the followup bot is working on the local Zulip instance,
+* Test manually that the helloworld bot is working on the local Zulip instance,
 by:
 
- - sending a few messages starting with `@followup` and `@follow-up` from
- different streams, such as `devel` and `social`
+ - sending a few messages starting with `@<your-name>_helloworld_bot` from
+ the stream you subscribed the bot to.
```
Comment (derAnfaenger) [inline tasks/2017/interactive-bots.md]: Same as above. Let's just say they should send the messages from "Verona".

File: tasks/2017/interactive-bots.md [hunk 3]
```diff
@@ -35,92 +35,56 @@ messages and can be integrated with various APIs.
...
 
- - checking that the messages showed up in the `followup` stream, prepended
- by the sender email
+ - checking that the bot is replying with `beep boop` message each time.
```
Comment (derAnfaenger) [inline tasks/2017/interactive-bots.md]: ~~message~~.

File: tasks/2017/interactive-bots.md [hunk 4]
```diff
@@ -35,92 +35,56 @@ messages and can be integrated with various APIs.
...
+`interactive bots: Run the helloworld bot for <username>.`.
 
-* Create a pull request in the `zulip/zulip-gci` repository, with title
-`interactive bots: Run the followup bot for <username>`. Link to your GCI
+* Create a pull request in the `zulip/zulip-gci-submissions-2017` repository, with title
+`interactive bots: Run the helloworld bot for <username>`. Link to your GCI
 task in comment section.
 
-*Completion criteria:* Mentors will check that the followup bot was properly
+*Completion criteria:* Mentors will check that the helloworld bot was properly
```
Comment (derAnfaenger) [inline tasks/2017/interactive-bots.md]: We should use consistent quoting for referring to the `helloworld` bot (sometimes it's "", sometimes '' and here without quotes). I'd just go without quotes everywhere.

File: tasks/2017/interactive-bots.md [hunk 5]
```diff
@@ -35,92 +35,56 @@ messages and can be integrated with various APIs.
...
-`interactive-bots/followup/<username>/`.
+`interactive-bots/helloworld/<username>/`.
 
 * Create a commit with the screenshots and notes, with commit message
-`interactive bots: Run the followup bot for <username>.`.
+`interactive bots: Run the helloworld bot for <username>.`.
 
-* Create a pull request in the `zulip/zulip-gci` repository, with title
-`interactive bots: Run the followup bot for <username>`. Link to your GCI
+* Create a pull request in the `zulip/zulip-gci-submissions-2017` repository, with title
```
Comment (derAnfaenger) [inline tasks/2017/interactive-bots.md]: zulip-gci-submissions ~~-2017~~

File: tasks/2017/interactive-bots.md [hunk 6]
```diff
@@ -35,92 +35,56 @@ messages and can be integrated with various APIs.
...
+ - checking that the bot is replying with `beep boop` message each time.
 
 * Take screenshots showing that the bot is working, make sure to have
 screenshots of:
 
  - your terminal window with the bot running, including the command you used
  to run the bot and the output with the bot description
- - messages sent by you to the bot in the `devel` and `social` streams
- - messages sent by the bot in the `followup` stream
+ - messages sent by you to the bot in the subscribed stream
```
Comment (derAnfaenger) [inline tasks/2017/interactive-bots.md]: "Verona"

File: tasks/2017/interactive-bots.md (hunk 1 above)
Comment (abhijeetkaur) [reply tasks/2017/interactive-bots.md]: Oh is it? That's interesting.
So, the bots get subscribed to all the streams or the ones that the owner of the bot is subscribed to?

File: tasks/2017/interactive-bots.md (hunk 4 above)
Comment (abhijeetkaur) [reply tasks/2017/interactive-bots.md]: Ah, interesting catch!

File: tasks/2017/interactive-bots.md (hunk 1 above)
Comment (derAnfaenger) [reply tasks/2017/interactive-bots.md]: No, the change is that bots don't need to be subscriobed to a stream to receive @-mentions anymore.

**Comment by derAnfaenger** [comment]
Merging this, thanks @abhijeetkaur! I'll do one more general update of this once #611 has been merged.

**Pull request merged**